ehitchhiking-ui/src/app/shared/components/chat-data/dialog-list/dialog-list.component.ts

```diff
@@ -9,6 +10,11 @@ import {DialogListService} from './dialog-list.service';
 export class DialogListComponent implements OnInit {
 	dialogList = DialogListService.dlgList;
 	@Output() chatMessages = new EventEmitter<ChatMessage[]>();
+	noDataSize: NoDataSize = NoDataSize.Small;
+	noDataMessage = 'No messages!';
+	noDataIconName = 'accessibility';
+	loading = false;
+	defaultImg = 'http://ust-barguzin.ucoz.ru/02-2016/polzovatel-izobrazhenie-s-chernym-fonom_318-34564..jpg';
```

Comment:
pls download this image and it to the project

Reply:
done